ethcore/light/src/lib.rs

```diff
@@ -63,7 +63,8 @@ extern crate parity_bytes as bytes;
 extern crate ethereum_types;
 extern crate ethcore;
 extern crate hash_db;
-extern crate heapsize;
+extern crate parity_util_mem as malloc_size_of;
+extern crate parity_util_mem as mem;
```

Comment:
I think this is confusing, it will require devs to come here to look. I think I'd prefer explicit paths here.